pages/returning-members.js

```diff
@@ -0,0 +1,282 @@
+import React, { useState, useContext, useEffect, useRef } from 'react';
+import { isEmail } from "validator";
+import { ThemeContext } from "styled-components";
+import PageTemplate from '../frontend/components/templates/PageTemplate';
+import { SystemComponent, SystemSpan } from '../frontend/components/atoms/SystemComponents';
+import Subtitle from "../frontend/components/atoms/Subtitle";
+import Card from '../frontend/components/atoms/Card';
+import FieldSection from "../frontend/components/molecules/Form/FieldSection";
+import Button from '../frontend/components/atoms/Button';
+
+const TERMDESCRIPTIONS = [
+"1A Co-op", 
+"1B Study", 
+"1B Co-op", 
+"2A Study", 
+"2A Co-op", 
+"2B Study", 
+'2B Co-op',
+"3A Study", 
+"3A Co-op",
+"3B Study", 
+"3B Co-op", 
+"4A Study",
+"4A Co-op", 
+"4B Study", 
+"4B Co-op", 
+"Graduate Student", 
+"Not currently enrolled at the University of Waterloo", 
+"Alumni",
+];
+
+const PREVTERMS = ['F22', 'S22', 'W22','F21', 'S21', 'W21', 'F20', "S20",'W20', 'F19', 'S19', 'W19', 'F18', 'S18', 'W18'];
+
+const FUTURETERMS = ['W22', 'S22', 'F22', 'W23', "S23",'F23', 'None of the above'];
+
+const SUBTEAMS = ["Business", "LIM", "Mechanical", "MC", "BMS", "Embedded","Infrastructure", "Web", "Team Hub"]
+
+const ACTIVESTATUS = [
+    "Yes, I will continue on the team, and I will be on campus (or working locally)", 
+    "Yes, I will continue on the team remotely and can come to campus if needed/possible", 
+    "Yes, I will continue on the team remotely only",
+    "Undecided or unsure", 
+    "No, taking the term off" 
+]
+
+const MEMBERSHIP = [
+    'Member', 
+    'Exec/lead/advisor', 
+    'Coop', 
+    'Not active on Waterloop this term',
+];
+
+const CONTINUINGSTATUS = [
+    'Continue with my sub-team', 
+    'Transfer to another sub-team (please specify)', 
+    'Want to take on a leadership role - lead', 
+    'Want to take on a leadership role - co-op supervisor', 
+    'Want to become a co-op', 
+    'I\'m undecided or not continuing',
+]
+
+const FormHeader = ({title, marginBottom}) => {
+    const theme = useContext(ThemeContext);
+    return (
+        <SystemComponent
+            fontSize={theme.fontSizes.header3}
+            textAlign='center'
+            mb={marginBottom}
+        >
+            <SystemSpan>
+                <Subtitle>{title}</Subtitle>
+            </SystemSpan>
+        </SystemComponent>
+    );
+}
+
+const RegistrationForm = () => {
+    const theme = useContext(ThemeContext);
+
+    const [formValues, setFormValues] = useState({
+        fullName:  "",
+        termDescription: "", 
+        previousTerms: [], 
+        futureTerms: [], 
+        subteam: "",
+        activeStatus: "", 
+        continuingStatus: "", 
+        email: "", 
+        comments: "",
+        futureTasks: "", 
+    });
+
+    const [hasError, setHasError] = useState({
+        fullName: false,
+        termDescription: false, 
+        membership: false, 
```

Comment:
I don't think membership is a field on this form, right ? Can you please double-check.

Reply:
yup membership is not a field thanks for pointing that out